index.html

```diff
@@ -123,6 +123,71 @@
         </div>
       </ul>
     </section>
+    <section class="Earnings__right-side" aria-label="Here you can enter all the requested capes">
+      <img class="Earnings__right-sign" src="image/Frame.png" alt="logo Ember">
+      <dl class="Earnings__right-sign-up">
+        <dt class="Earnings__right-title">Sign up for an account</dt>
+        <dd class="Earnings__right-text">Send, spend and save smarter</dd>
+      </dl>
```

Comment:
Pues esto mas que una definición parece es un titulo y un texto
```suggestion
      <h1 class="Earnings__right-title">Sign up for an account</h1>
      <p class="Earnings__right-text">Send, spend and save smarter</p>
```